### ISSUE TITLE
Usage of Logo + hopefully constructive critique

### ISSUE THREAD

**Issue by HeIIow2** [wasi-master/sponsorblock.py#14]
Hi,

I really appreciate your work, and I already contributed once to your project, but I seem to have overlooked some things. After consideration, I decided it would be less work, to implement my own [sponsorblock wrapper](https://gitea.elara.ws/music-kraken/python-sponsorblock) than to "_fix_" the issues in your project.

I got a segment in my readme, why I made this decision. If you want you can read this critique, but ofc you don't have to.

Regardless, I consider using your logo as logo for my wrapper. I can see why you might be opposed to this, thus I ask. Would it be fine with you if I used your logo in my own api wrapper, which kinda does the same? If you are opposed to this I ofc. won't do it.

I hope you have a nice day 
<333

**Comment by wasi-master** [comment]
Yeah, it's fine, you can use it. It's not that much of an original design anyway. And also yeah you changing some things based on your needs and publishing that separately is okay for me.
